packages/app/src/utils/explorers.ts

```diff
@@ -158,12 +158,21 @@ export const EXPLORERS_CONFIG: Record<NETWORK, ExplorerData> = {
   [NETWORK.PLASMA_TESTNET]: {
     networkExplorerName: "Plasmascan",
     networkExplorerUrl: "https://testnet.plasmascan.to/",
-    networkExplorerApiUrl: "https://testnet.plasmascan.to/api",
-    safeTransactionApi: "https://transaction-plasma-testnet.stage.safe.protofire.io/api",
+    networkExplorerApiUrl: "https://api.routescan.io/v2/network/testnet/evm/9746_5/etherscan/api",
+    safeTransactionApi: "https://transaction-plasma-testnet.safe.protofire.io/api",
     safeUrl: "https://app.safe.protofire.io/plasma-testnet",
```

Comment:
```suggestion
    safeUrl: "https://app.safe.protofire.io/plasma-testnet:",
```
Pls adjust this line as well